model/src/main/java/oracle/kubernetes/weblogic/domain/v2/BaseConfiguration.java

```diff
@@ -53,6 +54,21 @@
           + "Legal values are NEVER, ALWAYS, or IF_NEEDED.")
   private String serverStartPolicy;
 
+  /**
```

Comment:
The operator will restart the pods whenever the value is changed and there is no code enforcing that when the value changes, it must be incremented by one.

i.e. these comments do not accurately reflect how this field works.